Tests/configure_and_test_integration_instances.py

```diff
@@ -1679,16 +1681,47 @@ def update_integration_lists(new_integrations_names: List[str], packs_not_to_ins
     return list(set(new_integrations_names)), modified_integrations_names
 
 
-def get_packs_not_to_install(modified_packs_names: Set[str], build: Build) -> Set[str]:
+def get_packs_not_to_install(modified_packs_names: Set[str], build: Build) -> Tuple[Set[str], Set[str]]:
     """
     Return a set of packs to install only in the post-update.
     Args:
         modified_packs_names (Set[str]): The set of packs to install.
         build (Build): The build object.
     Returns:
         (Set[str]): The set of the packs names that supposed to be not installed in the pre-update.
+        (Set[str]): The set of the non hidden packs names.
     """
-    return get_turned_non_hidden_packs(modified_packs_names, build)
+    non_hidden_packs = get_turned_non_hidden_packs(modified_packs_names, build)
+    packs_with_higher_min_version = get_packs_with_higher_min_version(modified_packs_names - non_hidden_packs, build)
+    build.pack_ids_to_install = list(set(build.pack_ids_to_install) - packs_with_higher_min_version)
+    return packs_with_higher_min_version.union(non_hidden_packs), non_hidden_packs
```

Comment:
why do we return a tuple and not one set?

Reply:
We need not install the non-hidden and higher-version packs and only send only the non-hidden packs to the `get_changed_integrations` method.